scss/typography/_base.scss

```diff
@@ -203,6 +203,10 @@ $cite-font-size: rem-calc(13) !default;
 /// @type Color
 $cite-color: $dark-gray !default;
 
+/// Pseudo content for `<cite>` elements.
+/// @type String
+$cite-pseudo-content: '\2014 \0020';
```

Comment:
Append the **!default** annotation, and declare the same variable in `settings/_settings.scss` to allow anyone to override the default value.